From 18df8d1cb8ca74d9a4e5b7c13c5031eebfe6d4f0 Mon Sep 17 00:00:00 2001
From: Joyce Ooi <joyce.ooi@intel.com>
Date: Tue, 3 Nov 2020 15:52:59 +0800
Subject: [PATCH 141/151] Revert "HSD #1508459581: ARM: dts: adjust the clock
 frequency and falling time for i2c1"

commit  1f2f1ded05f1c6a1349fd0ffb20042679cd50d5e from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.64-lts

This reverts commit 40520ce1518d393efd70b6d80d4f19a1285129d6.
This change causes Arria10 LCD to fail although it fixes Arria10 display
port, so revert this change before finding a more appropriate solution.

Signed-off-by: Joyce Ooi <joyce.ooi@intel.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 arch/arm/boot/dts/socfpga_arria10_socdk.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi b/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
index 53906d4868fc..b3d99e434223 100644
--- a/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
+++ b/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
@@ -128,9 +128,9 @@
 	 * adjust the falling times to decrease the i2c frequency to 50Khz
 	 * because the LCD module does not work at the standard 100Khz
 	 */
-	clock-frequency = <50000>;
-	i2c-sda-falling-time-ns = <60000>;
-	i2c-scl-falling-time-ns = <60000>;
+	clock-frequency = <100000>;
+	i2c-sda-falling-time-ns = <6000>;
+	i2c-scl-falling-time-ns = <6000>;
 
 	lcd: lcd@28 {
 		compatible = "newhaven,nhd-0216k3z-nsw-bbw";
-- 
2.26.1

